bin/release.sh

```diff
@@ -26,9 +28,15 @@ export AMO_SECRET=$(curl ${secret_url} | jq ".secret.AMO_SECRET" -r)
 # Append a unique (at least if we don't push twice per minute to master branch...)
 # version prefix to the addon id as AMO requires new addon version for every upload
 # and also require it to be greater.
+# Note the `.` between the date and the time.
+# It is important as Services.vc, the module used by the add-on manager to compare
+# version strings ignores number that are too big! So split the date and the time.
+# (Services.vc.compare("44.0a1.201706030109", "44.0a1.201706031651") return 0 which means
+# the versions are identical...
 # /!\ this changes install.rdf without commiting it.
-VERSION_SUFFIX=$(date +%Y%m%d%H%M)
-sed -i -E 's/em:version=\"([0-9.ab-]+)\"/em:version=\"\1.'$VERSION_SUFFIX'\"/g' install.rdf
+VERSION_SUFFIX=$(date +%Y%m%d.%H%M)
```

Comment:
I had to change the version suffix to please the add-on manager. 
See the inlined comment.